<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentMainWindow</class>
 <widget class="QMainWindow" name="StudentMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="comparedLearningButton">
      <property name="text">
       <string>Compared Learning</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="freeTrainingButton">
      <property name="text">
       <string>&amp;Free Training</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>&amp;Exit</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <layout class="QGridLayout" name="poseLayout"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="historyButton">
      <property name="text">
       <string>My History Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
